src/script/interpreter.h

```diff
@@ -143,6 +143,10 @@ enum : uint32_t {
     // Making unknown public key versions (in BIP 342 scripts) non-standard
     SCRIPT_VERIFY_DISCOURAGE_UPGRADABLE_PUBKEYTYPE = (1U << 20),
 
+    // Making unsigned annexes non-standard
+    //
+    SCRIPT_VERIFY_DISCOURAGE_UNSIGNED_ANNEX = (1U << 21),
```

Comment:
Oh, since Bitcoin Core does *not* have this patch, let's use a totally different bit number for this to avoid merge conflicts in the future.

Maybe the highest bit?